### ISSUE TITLE
SIL Transcriber audio burrito

### ISSUE THREAD

**Issue by gtryus** [bible-technology/scripture-burrito#288]
As a user working on drafting orally, recordings are made in the ogg format (compressed) using SIL Transcriber. The user would like to export the final audio content to be consumed by other products accepting audio Scripture burritos. I have been told by the Render team that is also planning support for ogg that the quality and compression are better than for mp3 and of course it doesn't have the history of licensing issues that are associated with mp3.

**Comment by jonathanrobie** [comment]
I think it would be helpful if people representing multiple audio software
systems could agree on what belongs in the audio flavor.   If it's just a
single application, like SIL Transcriber, then an "x-" flavor will do the
trick, but I imagine these will be exchanged among multiple applications.

Who should review and agree to the specification of the standard audio
flavor?  Sounds like at least SIL Transcriber and Render.  Who else?

**Comment by gtryus** [comment]
yes, I wrote because Eric asked me to write it up for you. Currently the standard only allows mp3 and wav. It would be good if the standard could be extended to at least allow ogg but not necessarily require it. One could imagine consumers being Scripture App builder. Scripture web sites are probably handled by a partner organization. Also I think Unfolding Word is planning to consume burritos...

**Comment by jonathanrobie** [comment]
Ideally, I would like to require people to be able to read the core content
of a burrito.   If the audio can be in .ogg, I would probably like to see
us require consumers to be able to read it.  Otherwise, we are saying that
our exchange format can produce valid files that consumers who need audio
cannot read.

**Comment by gtryus** [comment]
Yes, I agree. I think it would be good for consumers to be able to consume it. I think using an open standard like .ogg leads to good consumer support. Do you have others involved in the discussion about audio burritos? Eric and Tim thought we might be the only ones who were discussing it and therefore it would make sense for us to do what works for us and then others to follow...
